processor/internal/transformer/destination_transformer/embedded/warehouse/benchmark/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.24.4-alpine3.22@sha256:68932fa6d4d4059845c8f40ad7e654e626f3ebd3706eef7846f319293ab5cb7a AS builder
+FROM golang:1.25.3-alpine3.22
```

Comment:
I know it's for benchmarks only but if possible we can reintroduce the shas anyway.